generators/app/tasks/gitInitialization.js

```diff
@@ -50,19 +50,52 @@ module.exports = function gitInitialization() {
                       { cwd: `${process.cwd()}/${this.projectName}` }
                     ],
                     context: this.options
-                  }).then(() =>
-                    // git push origin master
-                    runCommand({
-                      command: [
-                        'git',
-                        ['push', 'origin', 'master'],
-                        { cwd: `${process.cwd()}/${this.projectName}` }
-                      ],
-                      context: this.options
-                    }).then(() => {
-                      spinner.succeed('git ready');
+                  })
+                    .then(() => {
+                      spinner.stop();
+                      return this.prompt([
+                        {
+                          type: 'confirm',
+                          name: 'pushToMaster',
+                          message: 'Do you want to push to master?'
```

Comment:
Instead of asking: `push to master?` I'd change this to ask `which branch do you want to use?` and use master as default value.
That way the branch to use is customizable and we dont need ` if (!pushToMaster) {`